src/map/skill.cpp

```diff
@@ -12999,10 +12999,10 @@ int32 skill_castend_nodamage_id (struct block_list *src, struct block_list *bl,
 			clif_skill_nodamage(src, *bl, skill_id, skill_lv);
 			// Detonate RL_B_TRAP
 			if (pc_checkskill(sd, RL_B_TRAP))
-				map_foreachinallrange(skill_bind_trap, src, AREA_SIZE, BL_SKILL, src);
+				map_foreachinshootrange(skill_bind_trap, src, AREA_SIZE, BL_SKILL, src);
```

Comment:
Isn't this fix the opposite of what was reported? That it should ignore walls.

Reply:
In the issue it says "can not" ignore walls multiple times